bin/fpfs_process_sim.py

```diff
@@ -13,17 +13,18 @@
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
 # GNU General Public License for more details.
 #
-import os
 import gc
-import jax
-import fpfs
 import json
-import schwimmbad
-import numpy as np
-import astropy.io.fits as pyfits
+import os
 from argparse import ArgumentParser
 from configparser import ConfigParser
 
+import astropy.io.fits as pyfits
+import fpfs
```

Comment:
Any reason putting so many empty lines?
Why not use the old version which takes less line and looks better?

Reply:
Fixed so that it matches with the one from main branch